deployment/step7.md

```diff
@@ -0,0 +1,24 @@
+# Deploy Edge Node
+
+Modify the $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json file. 
+Change metadata.name to name of the edge node to deploy.
+
+`vim $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json`{{execute HOST2 }}
+
+Deploy node using the below command.
+
+_kubectl apply -f $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json -s cloudhuburl:8080_
+
+_Replace cloudhuburl with the ip of the machine where cloud is running(use ens3 inet addr)._
+
+Perform above two steps multiple times to create multiple nodes.
+
+You will get a message telling node created.
```

Comment:
Give an example of the message and please remove sentences having you as I had requested earlier

Reply:
Updated